experiments/n2v/bsd68/careamics/careamics-bsd68.py

```diff
@@ -33,106 +29,70 @@
 test_path = data_path / "test" / "images"
 gt_path = data_path / "test" / "gt"
 
-# Load training image for N2V
-train_files = list(train_path.glob("*.tiff"))
-if len(train_files) == 0:
-    raise FileNotFoundError(f"No training files found in {train_path}")
+train_path.mkdir(parents=True, exist_ok=True)
+val_path.mkdir(parents=True, exist_ok=True)
+test_path.mkdir(parents=True, exist_ok=True)
+gt_path.mkdir(parents=True, exist_ok=True)
 
-print(f"Loading training file: {train_files[0]}")
-train_image = tifffile.imread(train_files[0])[0]
-print(f"Training image shape: {train_image.shape}")
+#### training and validation data
+train_image = tifffile.imread(next(iter(train_path.rglob("*.tiff"))))[0]
+val_image = tifffile.imread(next(iter(val_path.rglob("*.tiff"))))[0]
 
-# Configure Noise2Void model
-print("Configuring Noise2Void model...")
+#### configuration
 config = create_n2v_configuration(
     experiment_name="n2v_BSD",
     data_type="array",
-    axes="YX",
+    axes="SYX",
     patch_size=(64, 64),
     batch_size=128,
     num_epochs=50,
     masked_pixel_percentage=0.2,
     struct_n2v_axis="none",
-    model_checkpoint={"save_top_k": 3, "monitor": "val_loss"},
+    model_params={
+        "num_channels_init": 32
+    },
+    optimizer_params={
```

Comment:
same here, `optimizer_params` and `lr_scheduler` are not valid parameters of that function! They probably should...

Reply:
If you deal with the issue, then better not remove the call from here since you are about to fix it!